crates/server/src/admin/service.rs

```diff
@@ -136,6 +137,10 @@ pub(crate) fn setup(
             "/contexts/:context_id/identities-owned",
             get(get_context_identities::handler),
         )
+        .route(
+            "/contexts/:context_id/aliases",
+            get(get_context_aliases::handler),
+        )
```

Comment:
please follow convention, all alias related functionality should be in alias.rs as [mentioned](https://github.com/calimero-network/core/pull/1276#discussion_r2108136761)